### ISSUE TITLE
ABORT: ASSERTION FAILED: cr != RT_ROUTED_TUNNEL

### ISSUE THREAD

**Issue by hwdsl2** [libreswan/libreswan#670]
Using Libreswan 4.6 with multiple IKEv2 clients. The following error (leading to Libreswan crashing) occurs frequently:

```
pluto[X]: "ikev2-cp"[6] x.x.x.x #11: STATE_V2_ESTABLISHED_IKE_SA: retransmission; will wait 16 seconds for response
pluto[X]: "ikev2-cp"[6] x.x.x.x #11: STATE_V2_ESTABLISHED_IKE_SA: retransmission; will wait 32 seconds for response
pluto[X]: "ikev2-cp"[6] x.x.x.x #11: STATE_V2_ESTABLISHED_IKE_SA: 60 second timeout exceeded after 7 retransmits.  No response (or no acceptable response) to our IKEv2 message
pluto[X]: "ikev2-cp"[6] x.x.x.x #11: liveness action - clearing connection kind CK_INSTANCE
pluto[X]: "ikev2-cp"[6] x.x.x.x #12: ESP traffic information: in=xKB out=xMB
pluto[X]: "ikev2-cp"[6] x.x.x.x #11: deleting state (STATE_V2_ESTABLISHED_IKE_SA) aged x.xs and NOT sending notification
pluto[X]: ABORT: ASSERTION FAILED: cr != RT_ROUTED_TUNNEL (unroute_connection() +1485 programs/pluto/kernel.c)
ipsec__plutorun: !pluto failure!:  exited with error status 134 (signal 6)
ipsec__plutorun: restarting IPsec after pause...
ipsec__plutorun: Starting Pluto
```

I believe this may be a bug in the assertion code. @cagney or @letoams Can you please take a look?

**Comment by bleve** [comment]
This has been already fixed with eb8f748581d1fad2ef147037d16643361a02607a in main.

**Comment by bleve** [comment]
Actually the fix is bec2eeaeac8644203d3af56b951a526b29d45eb6.

**Comment by hwdsl2** [comment]
Great, thank you for fixing this!
